app/src/main/java/com/bucketlist/dto/LocationDetails.kt

```diff
@@ -0,0 +1,4 @@
+package com.bucketlist.dto
+
+data class LocationDetails(val longitude: String, val latitude: String){
+}
```

Comment:
I created this  new data class called LocationDetails and to learn more about this watch : https://www.youtube.com/watch?v=VgZZemAwLTk

Reply:
As an empty class, this appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality.
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.